app/src/main/java/org/fossasia/susi/ai/skills/aboutus/AboutUsFragment.kt

```diff
@@ -37,11 +39,58 @@ class AboutUsFragment : Fragment() {
 
     @NonNull
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        about_susi.movementMethod = LinkMovementMethod.getInstance()
-        contributors_desc.movementMethod = LinkMovementMethod.getInstance()
-        susi_skill_cms_desc.movementMethod = LinkMovementMethod.getInstance()
-        susi_report_issues_desc.movementMethod = LinkMovementMethod.getInstance()
-        susi_license_info_desc.movementMethod = LinkMovementMethod.getInstance()
+        about_susi.setOnClickListener({
+            try {
+                var uri = Uri.parse(getString(R.string.url_about_susi))
+                var builder: CustomTabsIntent.Builder = CustomTabsIntent.Builder()
+                var customTabsIntent = builder.build()
+                customTabsIntent.launchUrl(context, uri)
+            } catch (e: Exception) {
+                Toast.makeText(context, getString(R.string.link_unavailable), Toast.LENGTH_SHORT).show()
+            }
+        })
+
+        contributors_desc.setOnClickListener({
+            try {
+            var uri = Uri.parse(getString(R.string.url_susi_contributors))
+            var builder: CustomTabsIntent.Builder = CustomTabsIntent.Builder()
+            var customTabsIntent = builder.build()
+            customTabsIntent.launchUrl(context, uri)
+        } catch (e: Exception) {
+            Toast.makeText(context, getString(R.string.link_unavailable), Toast.LENGTH_SHORT).show()
+        }})
+
+        susi_skill_cms_desc.setOnClickListener({
+            try {
+                var uri = Uri.parse(getString(R.string.url_susi_skill_cms))
+                var builder: CustomTabsIntent.Builder = CustomTabsIntent.Builder()
+                var customTabsIntent = builder.build()
+                customTabsIntent.launchUrl(context, uri)
+            } catch (e: Exception) {
+                Toast.makeText(context, getString(R.string.link_unavailable), Toast.LENGTH_SHORT).show()
+            }
+        })
+
+        susi_report_issues_desc.setOnClickListener({
+            try {
+                var uri = Uri.parse(getString(R.string.url_susi_report_issue))
+                var builder: CustomTabsIntent.Builder = CustomTabsIntent.Builder()
```

Comment:
Any specific reason to specify type here as there is no need for such in this place

Reply:
I think it's fine

Reply:
As kotlin gives support to not explicitly caste the type, so i don't think it is of any use here according to Kotlin guidelines. If there is a case to override the functions then it probably be helpful in place.